assets/js/googlesitekit/modules/datastore/modules.test.js

```diff
@@ -40,6 +42,7 @@ import FIXTURES, { withActive } from './__fixtures__';
 import { MODULES_SEARCH_CONSOLE } from '../../../modules/search-console/datastore/constants';
 import { CORE_USER } from '../../datastore/user/constants';
 import { DASHBOARD_VIEW_GA4 } from '../../../modules/analytics/datastore/constants';
+import { resetGlobals } from '../../../../../.storybook/utils/resetGlobals';
```

Comment:
Hmm... AFAICT, we don't currently have any cases where we're importing from Storybook files in a test file, although we do import various test utilities in story files.

In order to keep this cross-pollination one-way as it is now, it might be worth moving this `resetGlobals()` utility into a test utility area. Otherwise we open the door to having our tests and storybook files somewhat intertwined which feels like it could get a bit harder to manage over time.

Or, alternatively extract the specific part of `resetGlobals()` that these tests depend on into a separate test utility...

Reply:
Makes sense! I didn't notice this one when I picked it up.

But, actually, after updating the regular Jest resets for other tests, this isn't needed 👍🏻